### ISSUE TITLE
OS type not identified in preprocessing.

### ISSUE THREAD

**Issue by Onager** [log2timeline/plaso#928]
**Plaso version:**

1.5 RC1 

**Operating system Plaso is running on:**

Ubuntu 14.04LTS 64 bit

**Installation method:**

GIFT

**Description of problem:**

Originally reported in #924.
OS type is not identified in preprocessing.

**Debug output/tracebacks:**
```
***************************** System configuration *****************************
                Hostname : victoria
        Operating system : N/A
Operating system product : N/A
Operating system version : N/A
               Code page : cp1252
         Keyboard layout : N/A
               Time zone : UTC
```

**Source data:**
victoria-v8.sda1.img from a Honeynet forensics challenge here: https://www.honeynet.org/challenges/2011_7_compromised_server

**Comment by joachimmetz** [comment]
WIP #241

**Comment by joachimmetz** [comment]
I'll add an item to #241 to add support for this. The lack of this functionality is not a show stopper for the 1.5 release. Closing issue.

**Comment by Onager** [comment]
This still looks like a bug to me: 
https://github.com/log2timeline/plaso/blob/19ed3b5e47297bf36caa3a27a47ea248634e197e/plaso/preprocessors/interface.py#L201
and
https://github.com/log2timeline/plaso/blob/19ed3b5e47297bf36caa3a27a47ea248634e197e/plaso/preprocessors/interface.py#L235

Should have identified the image as OPERATING_SYSTEM_LINUX

**Comment by joachimmetz** [comment]
It does, added a debug line,
```
2016-08-14 19:01:51,419 [INFO] (MainProcess) PID:32649 <engine> OS: Linux
```

but it is not stored in the system configuration, hence not shows by pinfo

**Comment by joachimmetz** [comment]
As discussed we'll disable the pinfo functionality for now to prevent confusion and fix the underlying issue by finishing #241
